### ISSUE TITLE
x/telemetry/cmd/gotelemetry: view not showing charts

### ISSUE THREAD

**Issue by danp** [golang/go#65517]
Running:

```
go run golang.org/x/telemetry/cmd/gotelemetry@master view
```

Using Go tip and the charts are missing:

<img width="949" alt="Screenshot 2024-02-04 at 1 20 09 PM" src="https://github.com/golang/go/assets/2182/10bf896c-478b-4db2-b621-ed5e0728f30b">

I see this error in the console:

```
Uncaught TypeError: t.Counters is not iterable
    at C3 (index.min.js:61:17104)
    at window.onload (index.min.js:61:16536)
```

Attributed to here:

https://github.com/golang/telemetry/blob/8e316b9e70bbf5540b59bf45fc711be3393cd997/internal/content/gotelemetryview/index.ts#L82

Inspecting things in the console I see:

```
> Page.Charts.Programs[0]
{
    "ID": "charts:cmd/go",
    "Name": "cmd/go",
    "Counters": null,
    "Active": false
}
```

Perhaps related to recent telemetry additions to the go command in tip?

**Comment by dr2chase** [comment]
@golang/tools-team not sure what's going on here.
I tried the reproducer, and I saw charts in my browser (which is Chrome).

@danp, is the problem here that there was nothing in your browser and that the JS crashed in some way?  Which browser/version was this?  I am only guessing that there was a browser involved because I tried your reproducer, it opened a browser window, and there were charts in it, and you mentioned Javascript and missing charts.
